unlock-app/src/components/creator/lock/LocksByNetwork.tsx

```diff
@@ -34,9 +34,9 @@ export const LocksByNetwork: React.FC<LocksByNetworkProps> = ({
 
   const { isLoading, data: locks } = useQuery([owner], async () => {
     const items = Object.values(networks)
-      .filter(({ subgraphURI }) => !subgraphURI?.includes('localhost'))
-      .map(async ({ id, subgraphURI }) => {
-        graphService.connect(subgraphURI)
+      .filter(({ subgraph }) => !subgraph?.endpoint.includes('localhost'))
+      .map(async ({ id, subgraph }) => {
+        graphService.connect(subgraph?.endpoint)
         const locksByNetwork = await graphService.locksByManager(owner, id)
         return [id, locksByNetwork as any]
```

Comment:
```suggestion
        if (subgraph?.endpoint) {
            graphService.connect(subgraph?.endpoint)
            const locksByNetwork = await graphService.locksByManager(owner, id)
            return [id, locksByNetwork as any]
        }
```

Reply:
maybe we open a different PR for this type of check, before connect to `Apollo` we need to make sure that  the endpoint exists the prevent errors

Reply:
Yep check for URI thoroughly will make things safer.